market/mysterium/dto.go

```diff
@@ -24,6 +24,12 @@ type CreateIdentityRequest struct {
 	Identity string `json:"identity"`
 }
 
+// CreatePayoutInfoRequest represents JSON request for creating payout info request
+type CreatePayoutInfoRequest struct {
+	Identity   string `json:"identity"`
```

Comment:
You dont need to repeat `Identity` in payload because it's already in params

Reply:
removed